apps/sms/views/conversation/js/startup.js

```diff
@@ -0,0 +1,87 @@
+/* global LazyLoader,
+          MessageManager,
+          Navigation,
+          TimeHeaders,
+          Settings,
+          LocalizationHelper,
+          ConversationView,
+          Information,
+          Threads,
+          Utils,
+          InterInstanceEventDispatcher
+*/
+
+(function(exports) {
+  'use strict';
+
+  const DEFAULT_PANEL = 'composer';
+
+  const LAZY_DEPENDENCIES = [
```

Comment:
Tried to re-use this list and ```initHeaders``` in the main app startup - but it looked more like a mess for me - so I went "duplication" way here.